packages/origin-ui-core/src/components/devices/import/ImportDeviceModal.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Button, Modal, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { FormInput, FormSelect } from '../../Form';
+import { Form, Formik } from 'formik';
+import Close from '@material-ui/icons/Close';
+import { Upload } from '../../Upload';
+import { useTranslation } from '../../../utils';
+
+const useStyles = makeStyles({
+    modalContent: {
+        display: 'flex',
+        width: '100%',
+        height: '100%',
+        alignItems: 'center',
+        pointerEvents: 'none'
+    },
+    modalWindow: {
+        background: '#434343',
```

Comment:
Please, avoid using hard coded color values. 
For getting colors use `useOriginConfiguration()` - and get colors from there. 
To modify the color use the `LightenColor` function.
If you follow this strategy - your colors will be auto-adjusted on color-scheme change.

Reply:
replaced with config variable